pkg/api/controllers/log/websocket.go

```diff
@@ -148,6 +151,19 @@ func readJSONLog(ginCtx *gin.Context, logReader io.Reader) {
 
 func ReadServerLog(ginCtx *gin.Context) {
 	server := server.GetInstance(nil)
+	retryQuery := ginCtx.DefaultQuery("retry", "-1")
```

Comment:
Let's use `true` instead. Apologies. My initial idea was to have a retry number here but your idea is better.